src/test/osd/osd-scrub-repair.sh

```diff
@@ -17,6 +17,12 @@
 source $(dirname $0)/../detect-build-env-vars.sh
 source $CEPH_ROOT/qa/workunits/ceph-helpers.sh
 
+if [ `uname` = FreeBSD ]; then
+    use_bluestore=false
```

Comment:
please see http://docs.ceph.com/docs/master/rados/operations/erasure-code/#erasure-coding-with-overwrites for more details. and you are disabling all ec_overwrite tests, so i think `use_ec_overwrite` accompanied with comment would be better.